src/librustc_mir/interpret/snapshot.rs

```diff
@@ -330,18 +304,6 @@ struct FrameSnapshot<'a, 'tcx> {
     stmt: usize,
 }
 
-impl_stable_hash_for!(impl<> for struct Frame<'mir, 'tcx> {
-    body,
-    instance,
-    span,
-    return_to_block,
-    return_place -> (return_place.as_ref().map(|r| &**r)),
```

Comment:
I see the impl was here.

@oli-obk What is going on here? Is this ignoring part of the frame for hashing?

Reply:
Not sure what's going on here, but there's nothing being ignored. Maybe it's just some missing impls?

Reply:
It seems to me it's skipping the `layout` field of `PlaceTy`. Is that intentional? If so, could it be skipped on the impl for `PlaceTy`?